### DISCUSSION TITLE
Ubuntu installed through this does not handling arrow keys

### DISCUSSION THREAD

**Discussion by riaz7se** [evansm7/vftool#26] [Q&A]
I have installed ubuntu through Vftool, Ubuntu started up, however once logged into as root, arrows keys were not working.
whenever I pressed arrow keys, it displays ^[[A, B, C, D for each arrow key.

I tried mapping keys in /inputrc and few more workarounds still issue exists. 
default shell is bash and terminal it shows as vt220.

Could you please suggest on this

**Comment by tommythorn** [comment]
I recommend logging into the VM via ssh, there everything works.  I'm using this 24/7, but don't use the console from where I started the VM, only log in via ssh.

**Comment by riaz7se** [comment]
@tommythorn log in via ssh has no issues and works well. **Thanks a lot** for quick reply and solution.

However arrow keys issue mentioned above is seen when vm is started with -t 0 arguments and logged into terminal. (log in via ssh does not have issues)

**Reply by tommythorn** [reply]
Two points: I’m just a humble user of this myself and I also see some related terminal issues.  However I’m used to similar issues with QEMU and other VMs so I don’t worry about it.
